source/layouts/layout.erb

```diff
@@ -3,7 +3,7 @@
 <head>
   <meta charset="utf-8">
   <title><%= current_page.data.title || settings.site_title %></title>
-  <meta name="description" content="The next generation open source app hosting platform by Red Hat">
+  <meta name="description" content="<%= current_page.data.description || "The next generation open source app hosting platform by Red Hat" %>">
```

Comment:
For SEO purposes, and search engine results, I would think that an appropriate description be available on the downloads page, different from the description on the index page